packages/playwright-core/src/server/javascript.ts

```diff
@@ -64,9 +65,11 @@ export class ExecutionContext extends SdkObject {
   private _delegate: ExecutionContextDelegate;
   private _utilityScriptPromise: Promise<JSHandle> | undefined;
   private _contextDestroyedRace = new ScopedRace();
+  readonly worldName: string;
```

Comment:
why is it not types.World?

Reply:
Because it is wider than that, I'll rename